lib/repeatable.js

```diff
@@ -31,6 +31,7 @@ module.exports = function(Queue) {
     now = prevMillis < now ? now : prevMillis;
 
     const nextMillis = getNextMillis(now, repeat);
+    delete repeat.immediately;
```

Comment:
I am not sure about this. Why do we need to delete it?

Reply:
Because it will be re-used by the next job and that's not what I want. When the first job have been completed `nextJobFromJobData` is called but we need to plan the next job to the next interval not immediately.

Reply:
ok, but we should not mutate properties from the input object, that can potentially create side effects on the callers code

Reply:
OK so my first solution was better ? If we don't want to mutate so we need to add a separate flag on the prototype of `nextRepeatableJob` and all over the code ?

Reply:
I do not remember the previous solution :/.

Reply:
Just the one with saving immediately flag in the job object, before my refactor commit.